docs/data-management.rst

```diff
@@ -159,7 +165,7 @@ For HCS data, you can create new Screens and link Plates to these Screens.
        |image2d|
 
 .. warning::
-    **The ``Copy Link`` feature will not create a new independent copy of the Images.** There is no possibility to copy Images in OMERO at the moment in a way resulting in a new independent copy. The only thing that is copied during the actions above is the link between the Images and the Dataset. A single Image becomes linked to two Datasets.
+    **The ``Copy Link`` feature will not create a new independent copy of the Images.** The only way to create a fully independent copy in OMERO is to use the :ref:`Duplicate feature<Duplicatecli>`. The only thing that is copied during the actions above is the link between the Images and the Dataset. A single Image becomes linked to two Datasets.
```

Comment:
```suggestion
    **The ``Copy Link`` feature will only create new links between an Image and a Dataset, so that one Image becomes linked to multiple Datasets. This does not create a new independent copy of the Image.** The only way to create a fully independent copy in OMERO is to use the :ref:`Duplicate feature<Duplicatecli>`.
```
Note also that having shorter lines would be useful for review.